all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -57,7 +57,7 @@ objects:
       spec:
           containers:
             - name: ${JAEGER_NAME}
-              image: jaegertracing/all-in-one:f209812d
+              image: jaegertracing/all-in-one:9f3984cd
```

Comment:
This is the issue we are discussing, at the moment there is no jaeger version. 

> These templates do not directly depend on the exact versions of Jaeger backend

I think this does not hold always, e.g. recently REST endpoints have been added and this template does not expose them. I think there can be more incompatible issues.